### PR TITLE
Add a build pipeline for OpenShift Ansible

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @csrwng 
ready for review

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
artifacts: https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/logs/openshift_openshift-ansible/ci-openshift-ansible-build/55/artifacts/

File: jobs/openshift/openshift-ansible/build.groovy [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+library "github.com/stevekuznetsov/release-library@skuznets/build-pipeline"
+
+buildPipeline(
+  "build",
+  cloneStep("openshift-ansible:test-base"),
+  [buildStep(
+    "build-rpms",
+    "src", "rpms",
+    "RUN umask 0002 && mkdir -p /srv/openshift-ansible/noarch && tito tag --offline --accept-auto-changelog && tito build --output=/srv/ --rpm --test --offline --quiet && createrepo /srv/openshift-ansible/noarch"
```
Comment (stevekuznetsov) [inline jobs/openshift/openshift-ansible/build.groovy]: can we push this into the repo or is it short enough?

Comment (csrwng) [reply jobs/openshift/openshift-ansible/build.groovy]: seems short enough to me though I don't like everything in a single line... you could split it up in a multi-line literal

**Pull request merged**